packages/node/src/TestProvider.ts

```diff
@@ -23,6 +23,11 @@ export class TestProvider extends providers.BaseProvider {
     )
   }
 
+  getWalletForPublicKey (address: Address): Wallet | undefined {
```

Comment:
I'd rename it to `getWalletForAddress` since the address and the public key are different things

Reply:
good point 👍